cmd/opm/registry/serve.go

```diff
@@ -57,6 +58,23 @@ func runRegistryServeCmdFunc(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	db, err := sql.Open("sqlite3", dbName) // TODO: ?immutable=true
+	if err != nil {
+		return err
+	}
+
+	// Migrate database to latest version before serving
+	migrator, err := sqlite.NewSQLLiteMigrator(db, "")
+	if err != nil {
+		return err
+	}
+	defer migrator.CleanUpMigrator()
```

Comment:
probably shouldn't defer this, it won't get run until the server exits (which could be never)